src/components/AddressInput.js

```diff
@@ -177,11 +182,11 @@ class AddressInput extends React.Component {
                 className="w-100"
                 countries={countries}
                 id={stateId}
-                name={this.nameFor('state')}
+                name={inputName.state}
                 placeholder={labels.state}
-                {...this.propsFor('state')}
+                {...this.propsFor(inputName.state)}
                 invalid={!!error.state}
-                onBlur={() => onBlur('state')}
+                onBlur={() => onBlur(inputName.state)}
                 onChange={state => this.onChange({ state })}
```

Comment:
I think the `onChange` should change based on the custom names as well, it's not working now...

Reply:
Ooh yes - didn't notice that that there were two that weren't using the flow thing.